package.json

```diff
@@ -24,18 +24,21 @@
     "husky": "^8.0.2",
     "jest": "^27.5.1",
     "jest-extended": "^2.0.0",
+    "jest-sorted": "^1.0.15",
     "pg-format": "^1.0.4",
     "supertest": "^7.0.0"
   },
   "dependencies": {
     "dotenv": "^16.0.0",
     "express": "^4.19.2",
+    "og": "^0.0.2",
```

Comment:
Looks like an accidental dependency install? Definitely worth removing with `npm uninstall` it if it's not being used :)